addons/website_sale/static/src/js/website_sale_tour_shop_cart_recovery.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('website_sale.tour_shop_cart_recovery', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var localStorage = require('web.local_storage');
+    var rpc = require('web.rpc');
+    var tour = require('web_tour.tour');
+
+    var productName = "Acoustic Bloc Screens";
+    var orderIdKey = 'website_sale.tour_shop_cart_recovery.orderId';
+    var recoveryLinkKey = 'website_sale.tour_shop_cart_recovery.recoveryLink';
+
+    tour.register('shop_cart_recovery', {
+        test: true,
+        url: '/shop?search=' + productName,
+        wait_for: base.ready(),
+    },
+    [
+        {
+            content: "select " + productName,
+            trigger: '.oe_product_cart a:containsExact("' + productName + '")',
```

Comment:
Not sure it is a good idea to compute trigger and content using variables. In this case, this does not increase readability and if the step fails at some point, people will have a hard time finding the step `.oe_product_cart a:containsExact("Acoustic Bloc Screens")`

Reply:
Good point, I'll change to help grepping.